shell/platform/windows/text_input_manager.h

```diff
@@ -0,0 +1,40 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_WINDOWS_TEXT_INPUT_MANAGER_H_
+#define FLUTTER_SHELL_PLATFORM_WINDOWS_TEXT_INPUT_MANAGER_H_
+
+#include <string>
+#include <Windows.h>
+#include <Windowsx.h>
+
+#include "flutter/shell/platform/common/cpp/geometry.h"
+
+namespace flutter {
+
+class TextInputManager {
+ public:
+  TextInputManager() noexcept = default;
+
+  void CreateImeWindow(HWND hwnd);
```

Comment:
Is there a reason for the HWND being a param to every single method, rather than to the constructor and stored in an ivar? (Or if constructor doesn't work, a `SetWindow` call that can be used when configuring it for an input session)

Reply:
For multi-window support, the same IME is used with multiple windows. That said, we can deal with it when we get there rather than me speculating on the right approach here. Done.